R/preview-standalone.R

```diff
@@ -436,3 +436,83 @@ st_image_show <- function(path,
   )
   return(ans)
 }
+
+#' Render and save a table in png or pdf format
+#'
+#' @param x an stable object.
+#' @param file the final output file name; if `file` has `.tex` extension, the
+#' extension will be replaced with the value in `format`.
+#' @param stem used to form the final output file name; this argument will
+#' override what is provided in `file`.
+#' @param format the output file format.
+#' @param dir the final output directory.
+#' @param build_dir the directory where the image will be built.
+#' @param ... passed to [st_aspdf()] or [st_aspng()], depending on the value of
+#' `format`.
+#'
+#' @details
+#' The table will be built / rendered in `tempdir()`.
```

Comment:
This seems confusing given that it depends on the `build_dir` argument. I think it'd be fine to drop this line.

Reply:
Thanks; this was left over from previous iteration where we didn't give `build_dir` option.